webblog/settings/base.py

```diff
@@ -112,4 +121,29 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
-STATIC_URL = '/static/'
+# STATIC_URL = '/static/'
+
+CORS_ORIGIN_ALLOW_ALL = True
+
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, "media")
+
+# STATICFILES_DIRS = [
+#     os.path.join(BASE_DIR, './static'),
+# ]
+
+AWS_ACCESS_KEY_ID = config('AWS_ACCESS_KEY_ID')
```

Comment:
Move this to production.
The uploaded images/file should be stored and served by django locally and for production they should be stored and served from AWS S3.

Reply:
Separated media storage for local and production.